src/Moonglade.Web/wwwroot/manifest.json

```diff
@@ -1,5 +1,7 @@
 {
- "name": "App",
+ "short_name": "Edi",
+ "name": "Edi.Wang",
```

Comment:
Please do not use any "Edi.Wang" specific code, because the final goal is to build a general blog system for anyone.